docs/metrics.md

```diff
@@ -31,25 +32,30 @@ All VMI metrics listed below contain, but are not limited to, these three labels
 * `node` - Node where the VMI is running on.
 
 #### kubevirt_vmi_memory_resident_bytes
+#### HELP kubevirt_vmi_memory_resident_bytes resident set size of the process running the domain.
```

Comment:
I wonder: why do is the name of the metric(`kubevirt_vmi_memory_resident_bytes`) is always repeated in the help message?